src/test/unit_tests/test_nc_health.js

```diff
@@ -693,6 +695,83 @@ mocha.describe('nsfs nc health', function() {
+        fs_utils.folder_delete(tmp_lifecycle_logs_dir_path);
+        await config_fs.delete_config_json_file();
+    });
+
+    mocha.it('Health lifecycle - lifecycle worker never ran on host', async function() {
+        try {
+            await nb_native().fs.readdir(config_fs.fs_context, tmp_lifecycle_logs_dir_path);
+            assert.fail('tmp_lifecycle_logs_dir_path should not exist');
+        } catch (err) {
+            assert.equal(err.code, 'ENOENT');
+        }
+        const health_status = await Health.nc_nsfs_health();
+        assert_lifecycle_status(health_status, {});
+    });
+
+    mocha.it('Health lifecycle', async function() {
+        await exec_manage_cli(TYPES.LIFECYCLE, '', { config_root, disable_service_validation: true, disable_runtime_validation: true }, true);
+        const lifecycle_log_entries = await nb_native().fs.readdir(config_fs.fs_context, tmp_lifecycle_logs_dir_path);
+        assert.strictEqual(lifecycle_log_entries.length, 1);
+        const log_file_path = path.join(tmp_lifecycle_logs_dir_path, lifecycle_log_entries[0].name);
+        const lifecycle_log_json = await config_fs.get_config_data(log_file_path, { silent_if_missing: true });
+        const health_status = await Health.nc_nsfs_health();
+        assert_lifecycle_status(health_status, lifecycle_log_json);
+    });
+
+    mocha.it('Health lifecycle - should report on timeout error', async function() {
+        await config_fs.update_config_json_file(JSON.stringify({
+            NC_LIFECYCLE_TIMEOUT_MS: 1,
+            NC_LIFECYCLE_LOGS_DIR: tmp_lifecycle_logs_dir_path
+        }));
+        await exec_manage_cli(TYPES.LIFECYCLE, '', { config_root, disable_service_validation: true, disable_runtime_validation: true }, true);
+        const lifecycle_log_entries = await nb_native().fs.readdir(config_fs.fs_context, tmp_lifecycle_logs_dir_path);
+        assert.strictEqual(lifecycle_log_entries.length, 1);
+        const log_file_path = path.join(tmp_lifecycle_logs_dir_path, lifecycle_log_entries[0].name);
+        const lifecycle_log_json = await config_fs.get_config_data(log_file_path, {silent_if_missing: true});
+        const health_status = await Health.nc_nsfs_health();
+        assert_lifecycle_status(health_status, lifecycle_log_json);
+    });
+
+    mocha.it('Health lifecycle - run lifecycle 3 times - should report on latest run', async function() {
+        this.timeout(5000);// eslint-disable-line no-invalid-this
+        await exec_manage_cli(TYPES.LIFECYCLE, '', { config_root, disable_service_validation: true, disable_runtime_validation: true }, true);
+        await exec_manage_cli(TYPES.LIFECYCLE, '', { config_root, disable_service_validation: true, disable_runtime_validation: true }, true);
+        const latest_lifecycle = await exec_manage_cli(TYPES.LIFECYCLE, '', { config_root, disable_service_validation: true, disable_runtime_validation: true }, true);
+        const parsed_res = JSON.parse(latest_lifecycle);
```

Comment:
I think it can be easier to understand with additional information in the variable name.
```suggestion
        const parsed_res_latest_lifecycle = JSON.parse(latest_lifecycle);
```